.travis.yml

```diff
@@ -11,10 +11,10 @@ python:
 - "nightly"
 
 .mixtures:
-- &py3_7-base
-  python: "3.7"
-  dist: xenial
-  sudo: required
+  - &py3_7-base
```

Comment:
please move it back. extra indentation is not needed

Reply:
ok. I will correct